http-server-jetty/src/main/java/io/micronaut/servlet/jetty/JettyFactory.java

```diff
@@ -167,6 +191,20 @@ protected Server jettyServer(
         return server;
     }
 
+    /**
+     * @param jettyRequestLog the jetty request log configuration.
+     * @return {@link RequestLog} logback-access impl.
+     */
+    @Singleton
+    @Requires(property = JettyConfiguration.JettyRequestLog.ENABLED_PROPERTY, value = StringUtils.TRUE)
+    RequestLog requestLog(JettyConfiguration.JettyRequestLog jettyRequestLog) {
```

Comment:
this should be split into a separate factory with a `@Requires(classes= RequestLogImpl.class)` definition on it. The default should be kept and logback-access only activated when it is on the classpath